src/components/utils/src/threads/async_runner.cc

```diff
@@ -54,12 +54,12 @@ void AsyncRunner::AsyncRun(ThreadDelegate* delegate) {
 
 void AsyncRunner::Stop() {
   SDL_AUTO_TRACE();
-  thread_->join();
+  thread_->join(true);
 }
 
 AsyncRunner::~AsyncRunner() {
   SDL_AUTO_TRACE();
-  thread_->join();
+  thread_->join(true);
   delete executor_;
```

Comment:
@wolfylambova22, this update is related only with the join() method - I've added a parameter. No need of additional calls because of this.